package.json

```diff
@@ -21,15 +22,16 @@
     "graphiql": "^0.17.5",
     "graphql": "^14.6.0",
     "graphql-relay": "^0.6.0",
+    "install": "^0.13.0",
     "js-base64": "^2.5.2",
+    "npm": "^6.14.1",
```

Comment:
Why is this being added?

Reply:
Probably npm install added it. Will remove